fabfile/use.py

```diff
@@ -71,3 +79,11 @@ def asserts(*args):
 def debug_mode(*args):
     '''ps's configure is passed: '--enable-debug --enable-cassert CFLAGS="-ggdb -Og -g3 -fno-omit-frame-pointer"' '''
     config.PG_CONFIGURE_FLAGS.append('--enable-debug --enable-cassert CFLAGS="-ggdb -Og -g3 -fno-omit-frame-pointer"')
+
+
+@task
+def valgrind(*args):
+    config.PG_CONFIGURE_FLAGS.append('--enable-cassert --enable-debug --with-uuid=e2fs CFLAGS="-ggdb -Og -DUSE_VALGRIND"')
+
+    # set this flag to trigger libuuid-devel installation
+    config.settings['is_valgrind_test'] = True
```

Comment:
this variable is a leftover I think, it seems that it is not used anywhere.

Reply:
yep, it is removed